app/views/pattern_library/index.slim

```diff
@@ -51,13 +51,13 @@ h2 Molecules
   .pl-heading
     h3 Dropdown Menu
   .pl-preview
-    == react_component 'dropdownMenu', { item: 'Alpha', selected: 'selected', name: 'Some', url: '1'}
+    == react_component 'DropdownMenu', { item: 'Alpha', selected: 'selected', name: 'Some', url: '1'}
 
 .pl-group
   .pl-heading
     h3 Dropdown Menu Multiselect
   .pl-preview
-    == react_component 'dropdownMenuMultiSelect', { selected: 'false', name: 'Alpha', item: 'Some' }
+    == react_component 'DropdownMenuMultiSelect', { selected: 'false', name: 'Alpha', item: 'Some' }
```

Comment:
@slowbot this is my bad coz I didn't notice this in the PL PR but our current React naming conventions are that we also capitalize the first letter of the component - for future PL component stories.